Kickstarter-iOS/Views/AmountInputView.swift

```diff
@@ -0,0 +1,119 @@
+import Library
+import Prelude
+import UIKit
+
+class AmountInputView: UIView {
+  // MARK: - Properties
+
+  private lazy var label: UILabel = { UILabel(frame: .zero) }()
+  private lazy var textField: UITextField = { UITextField(frame: .zero) }()
+  private lazy var stackView: UIStackView = { UIStackView(frame: .zero) }()
+  private var labelCenterYAnchor: NSLayoutConstraint?
+
+  // MARK: - Lifecycle
+
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+
+    self.stackView.addArrangedSubview(self.label)
+    self.stackView.addArrangedSubview(self.textField)
+
+    self.addSubviewConstrainedToEdges(self.stackView)
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+    super.traitCollectionDidChange(previousTraitCollection)
+
+    let traitCollection = self.traitCollection
+
+    if previousTraitCollection?.preferredContentSizeCategory
```

Comment:
Does this not work if you just put it in `bindStyles`? I believe that is called when the traitcollection changes.

Reply:
Yeah, it works without it thanks to our swizzling....why did I do this Justin? 🤦‍♂️